Dockerfile

```
@@ -1,6 +1,9 @@
 FROM nginx:1.14-alpine
 
+RUN apk add jq
```

```suggestion
RUN apk --no-cache add jq
```

For a smaller resulting image